### PR TITLE
v18 Continuum opening campaign

### PR DESCRIPTION
**Pull request by flyingrobots**
## Summary
- Harden Continuum artifact ingestion around real Continuum fixture JSON and Wesley realization manifest JSON.
- Require explicit load context for artifact authority, rejecting self-attested `authority` fields and local/handwritten mirror authority.
- Update BEARING and cycle docs for the non-prefixed branch and completed task 5 evidence.

Supersedes #90; this PR replaces the prefixed branch with `v18-continuum-opening`.

## Validation
- `npm run lint`
- `npm run typecheck:src`
- `npm run typecheck:test`
- `npm run typecheck:surface`
- `npm run typecheck:policy`
- `npm run lint:sludge`
- `npm run lint:quarantine-graduate`
- `npx --yes markdownlint-cli docs/BEARING.md docs/design/0146-v18-continuum-compatibility-charter/v18-continuum-compatibility-charter.md docs/design/0149-v18-continuum-artifact-ingestion/v18-continuum-artifact-ingestion.md`
- `npx vitest run test/unit/domain/continuum/ContinuumArtifactIngestionPolicy.test.ts test/unit/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.test.ts test/unit/domain/index.exports.test.ts test/unit/domain/errors/index.test.ts`
- `npm run test:local` (447 files, 6821 tests)
- `npm run test:coverage:ci` (92.14% lines; threshold 91.90%)
- Pre-push IRONCLAD M9 gates passed

## Summary by CodeRabbit

* **New Features**
  * Adds Continuum artifact compatibility: family identifiers, authority model, descriptors, ingestion policy, and a JSON adapter to load/validate/ingest artifacts and manifests.

* **Documentation**
  * v17 closeout/retro updates and extensive v18 Continuum design docs (charter, contract matrix, optic mapping, ingestion slices) plus backlog/method/vision framing refreshes.

* **Tests**
  * New fixtures and comprehensive unit tests covering descriptor, policy, adapter parsing and validation.

[Review Change Stack](https://app.coderabbit.ai/change-stack/git-stunts/git-warp/pull/91?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

**Comment by flyingrobots** [comment]
@codex Self-review pass after clean-worktree check and `git fetch origin`.

Issue count: Critical 0, Major 3, Minor 2.

| Severity | File / lines | Infraction | Evidence | Recommended mitigation prompt |
| --- | --- | --- | --- | --- |
| Major | `src/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.ts:166-177`, `src/domain/continuum/ContinuumArtifactIngestionPolicy.ts:13-15` | Shape and authority are not bound together. | The adapter always copies `context.authority` into the descriptor, and the policy accepts either `generated-artifact` or `generated-fixture` for every descriptor. A Wesley realization manifest can therefore be admitted as a generated fixture, and a Continuum fixture can be admitted as a generated artifact. That weakens the branch goal of making generated-family authority explicit and non-folkloric. | `Update ContinuumArtifactJsonFileAdapter so parsed artifact shape determines the allowed authority: Wesley realization manifests must require generated-artifact, Continuum family fixtures must require generated-fixture, and mismatched context authority must throw AdapterValidationError. Add regression tests for both mismatches.` |
| Major | `src/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.ts:127-135`, `src/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.ts:158-162` | Caller context can override parsed artifact kind. | `context.artifactKind` can replace the canonical parsed kind for both Wesley manifests and Continuum fixtures. This lets an adapter caller emit a descriptor whose `artifactKind` contradicts the JSON shape the adapter just parsed. | `Remove artifactKind from ContinuumArtifactJsonLoadContext, or require it to exactly match the parsed shape before descriptor construction. Add tests proving a Wesley manifest cannot be labeled continuum.family.fixture and a Continuum fixture cannot be labeled wesley.realization.manifest.v1.` |
| Major | `src/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.ts:257-281`, `src/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.ts:284-302` | Wesley generated inventory can be empty or internally inconsistent. | `readGeneratedLegs` accepts `{}` because it freezes whatever keys exist and never requires at least one leg. `readOptionalNumber(leg, artifactCount)` validates only numeric shape; it does not require `artifactCount === files.length`, so a sealed manifest can claim generated artifacts without a matching file inventory. | `Harden Wesley generatedLegs parsing: require at least one generated leg, require artifactCount to equal the number of file entries when present, and reject positive artifactCount with missing or empty files. Add tests for empty generatedLegs, count mismatch, and positive count without files.` |
| Minor | `test/fixtures/continuum/receipt-family-wesley-realization-manifest.json:4` | Fixture commits a machine-specific absolute path. | The fixture stores `~/git/continuum/...` in `canonicalSchemaPath`. This is not used by the adapter, but it bakes a local workstation path into a shared repo fixture and makes the test data look less portable than the parser actually requires. | `Replace the absolute canonicalSchemaPath fixture value with a repo-neutral path or remove the optional field from the fixture if this test does not exercise it. Keep one parser test that validates canonicalSchemaPath as an optional string without committing a user-local path.` |
| Minor | `docs/design/0149-v18-continuum-artifact-ingestion/v18-continuum-artifact-ingestion.md:104-116` | Verification prose records a command known to exit nonzero as an observed coverage result. | The doc lists `npx vitest run --coverage ...` and then states that the command exits nonzero because global thresholds apply to the subset. That is accurate but easy to misread as a passing verification command in a completed cycle doc. | `Move the targeted coverage command under a clearly labeled diagnostic coverage note section, or replace it with a command/configuration that exits zero while reporting only touched-file coverage. Keep the full passing coverage CI command as the authoritative verification line.` |

**Comment by flyingrobots** [comment]
@coderabbitai review please

File: src/domain/continuum/ContinuumArtifactIngestionPolicy.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  /** Rejects descriptors whose authority would make local mirrors canonical. */
+  assertGeneratedAuthority(descriptor: ContinuumArtifactDescriptor): void {
+    if (descriptor.hasGeneratedAuthority()) {
+      return;
+    }
+    throw new ContinuumArtifactAuthorityError(
+      `Continuum family ${descriptor.familyId.toString()} must be loaded from a generated artifact or fixture, not ${descriptor.authority.toString()}`,
+    );
```
[the remark this answers is not in the document]
Comment (flyingrobots) [reply src/domain/continuum/ContinuumArtifactIngestionPolicy.ts:26]: @coderabbitai Resolved in fbd240f5. The ingestion policy now computes the expected generated authority from the descriptor artifact kind and rejects mismatches such as `continuum.family.fixture` + `generated-artifact` or `wesley.realization.manifest.v1` + `generated-fixture`. The same commit adds regression coverage for both mismatch directions and for unknown generated artifact kinds.

**Comment by flyingrobots** [comment]
@coderabbitai Review feedback resolution summary:

| Issue | Severity | Resolution | Commit |
|---|---:|---|---|
| Policy accepted generated authority without checking descriptor kind pairing | Major | `ContinuumArtifactIngestionPolicy` now maps artifact kind to the required generated authority and rejects mismatches or unknown generated shapes; added regression coverage. | fbd240f5 |
| Policy test used environment-shaped `~/git/continuum/...` schema paths | Minor | Replaced repeated local path literals with `RECEIPT_SCHEMA_PATH`, a repo-neutral fixture constant. | fbd240f5 |
| `ContinuumArtifactJsonFileAdapter.ts` exceeded source-size/cohesion policy | Major | Split file entry point from JSON parsing, validation, Continuum fixture validation, and Wesley inventory validation modules; adapter entry point is now 34 LOC. | 6e529d8b |
| Verification docs and changelog needed to reflect review fixes | Documentation | Updated CHANGELOG, BEARING, cycle 0149 focused counts, and full coverage result. | 94226019, 7728214a |

Local verification before push:

```text
npm run lint
npm run typecheck:src
npm run typecheck:test
npm run typecheck:surface
npm run typecheck:policy
npm run lint:sludge
npm run lint:quarantine-graduate
npm run lint:md
git diff --check
npx vitest run test/unit/domain/continuum/ContinuumArtifactIngestionPolicy.test.ts test/unit/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.test.ts test/unit/domain/index.exports.test.ts test/unit/domain/errors/index.test.ts
npm run test:local
npm run test:coverage:ci
```

Observed full-suite results: `6826` tests passed; coverage gate passed at `92.13%` line coverage. Pre-push IRONCLAD also passed and pushed these commits.

**Comment by flyingrobots** [comment]
@coderabbitai Review follow-up pushed.

| Finding | Resolution | Commit |
|---|---|---|
| `src/infrastructure/adapters/continuumArtifactJsonTypes.ts` grouped peer exported types. | Split each exported type into its own adapter-local type file and rewired imports directly to the dedicated files. | `4725ed4e` |
| `test/unit/domain/continuum/ContinuumArtifactIngestionPolicy.test.ts` repeated authority literals. | Added named authority fixture constants and replaced inline authority values in the touched policy tests. | `b0b7bf96` |
| Review-response documentation. | Recorded the follow-up in `CHANGELOG.md` and `docs/BEARING.md`. | `040008a4` |

Local validation before push:

- `npm run lint`
- `npm run typecheck:src`
- `npm run typecheck:test`
- `npm run typecheck:surface`
- `npm run typecheck:policy`
- `npm run lint:sludge`
- `npm run lint:quarantine-graduate`
- `npm run lint:md`
- `git diff --check`
- `npx vitest run test/unit/domain/continuum/ContinuumArtifactIngestionPolicy.test.ts test/unit/infrastructure/adapters/ContinuumArtifactJsonFileAdapter.test.ts test/unit/domain/index.exports.test.ts test/unit/domain/errors/index.test.ts`
- `npm run test:local` (447 files, 6826 tests)
- `npm run test:coverage:ci` (447 files, 6826 tests, 92.13% line coverage)

Pre-push hook also passed link check, static gates, and `npm run test:local`.

@coderabbitai review please
